build.sbt

```diff
@@ -5,13 +5,13 @@ lazy val namedcap = (project in file(".")).
   settings(
     name := "namedcap",
     organization := "com.github.tarao",
-    version := "0.0.2",
-    scalaVersion := "2.11.11",
-    crossScalaVersions := Seq("2.11.11", "2.12.3"),
+    version := "1.0.0",
```

Comment:
I incremented major version because support for major Scala versions is changed.